engine/swarm/secrets.md

```diff
@@ -846,3 +846,80 @@ the information from a Docker-managed secret instead of being passed directly.
 >**Note**: Docker secrets do not set environment variables directly. This was a
 conscious decision, because environment variables can unintentionally be leaked
 between containers (for instance, if you use `--link`).
+
+## Using Docker Secrets in Compose
```

Comment:
Two nitpicks:
* We use the normal form. `Use` instead of `Using`
* We only capitalize the first word and product names, and in this case the word "Docker" is not really needed. So it can be `Use secrets in Compose files`